src/components/timeseries/TimeSeries.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { TIMESERIES_DASHBOARD_URL } from '../../constants';
+import ErrorMessageWithIcon from '../error/ErrorMessageWithIcon';
+
+import ScreenStatusMessage from './ScreenStatusMessage';
+
+class TimeSeries extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            maintenanceMode: this.props.maintenanceMode
+        };
+    }
+
+    render() {
+        return (
+            (!this.state.maintenanceMode) ?
+                <div>
+                    <ScreenStatusMessage maintenanceMode={this.state.maintenanceMode} />
```

Comment:
```suggestion
                    <ScreenStatusMessage />
```

`this.state.maintenanceMode` will always be `false` here.
Also, ScreenStatusMessage does not seem to receive any props.

Reply:
Right. I was first sending the maintenance mode to this component and let it handle it. But I changed the approach but forgot to remove this part. I will update on the next commit.